chart/values.yaml

```diff
@@ -209,6 +209,9 @@ networkConfig:
   #     DK1:
   #       issuerKeys:
   #       - publicKey: "Ay02vkc6FGV8FwtvVsmBO2p7UdbZIcFhvMGFB40D3DKX"
+  #   stamps:
+  #     Denmark:
+  #       url: "https://stamp.narnia.example.com",
```

Comment:
I would have placed the url as part of the issuer, under DK1. Since the issuer can only withdraw what they have issued them selves anyway.